docs/conversion/sakai_12_mysql_conversion.sql

```diff
@@ -810,3 +810,11 @@ ALTER TABLE SAKAI_CONFIG_ITEM MODIFY VALUE varchar(3000);
 -- SAK-40528 Add index on ASN_ASSIGNMENT.CONTEXT
 ALTER TABLE ASN_ASSIGNMENT ADD INDEX `IDX_ASN_ASSIGNMENT_CONTEXT` (`CONTEXT`);
 -- END SAK-40528
+
+-- SAK-41207 Add indexes
+CREATE INDEX UK_hyk73ocki8gwvm3ajf8ls08ac ON ASN_ASSIGNMENT_ATTACHMENTS (ASSIGNMENT_ID);
```

Comment:
Existing table already has a key with a different name 

`CREATE TABLE `ASN_ASSIGNMENT_ATTACHMENTS ` (
  `ASSIGNMENT_ID` varchar(36) NOT NULL,
  `ATTACHMENT` varchar(1024) DEFAULT NULL,
  KEY `FK_HYK73OCKI8GWVM3AJF8LS08AC` (`ASSIGNMENT_ID`),
  CONSTRAINT `FK_HYK73OCKI8GWVM3AJF8LS08AC` FOREIGN KEY (`ASSIGNMENT_ID`) REFERENCES `asn_assignment` (`ASSIGNMENT_ID`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;`

Reply:
Right on MySQL these indexes were implicitly created, good catch!

Renaming an index is only available on MySQL 5.7+, hence the DROP then CREATE.